src/services/orders.js

```diff
@@ -0,0 +1,97 @@
+const { Orders } = require('../models');
+const { OrderStatuses } = require('../models');
+
+class OrdersService {
+  async createOrder (req, res) {
+    try {
+      const orderStatus = await OrderStatuses.findOne({
+        name: 'In progress'
+      }).exec();
+      const order = new Orders(req.body);
+      order.status = orderStatus._id;
```

Comment:
You can set 'req.body.status' to orderStatus._id and then initiate new Orders(req.body).

In this case there will be no need for line 12.

Reply:
Ok, but you can add it this way.

Reply:
I don't find the logic in that